src/components/photo_input.tsx

```diff
@@ -147,6 +148,7 @@ const PhotoInput: React.FC<PhotoInputProps> = ({
                     </Collapsible>
                     <input
                         ref={ref}
+                        data-testid="photo-input"
```

Comment:
Instead of adding the `data-testid` prop, could you find this element with a CSS selector, e.g., `button.btn.btn-outline-primary`?